### ISSUE TITLE
[REQUEST]: Ability to set custom percentages

### ISSUE THREAD

**Issue by MagicalNET** [deiteris/voice-changer#177]
### In a few words, describe your idea

Percentages

### More information

I think this would be a great idea, since I noticed it's very hard getting that right number% to what you'd like it to be at. This way you can have any value from 0-1000% you'd like. This should also be applied to chunk. Not sure if these hardcoded numbers need to be fixed, but thought I'd ask.

**Comment by deiteris** [comment]
Hi @MagicalNET, could you please elaborate a bit what you mean by custom percentages exactly? There's an idea to introduce an input field that would allow to enter an arbitrary value instead of slider. Is this what you mean?

**Comment by MagicalNET** [comment]
> Hi @MagicalNET, could you please elaborate a bit what you mean by custom percentages exactly? There's an idea to introduce an input field that would allow to enter an arbitrary value instead of slider. Is this what you mean?

Yep! That would be the idea. Then anyone can have any value they wish to have.. more customizable is always welcomed, instead of hard coded numbers. So for example, on the volume input, I can double click the number and set the value to what i want. But should keep the general look the same with the sliders, etc. But just open the ability to double click any of the numbers and set custom values.

**Comment by MagicalNET** [comment]
I'm going toss this in here, I noticed the volume level of speaking through a mic doesnt come out 1:1, is there a way to achieve 1:1? I'd figure using 100% on the voice changer would be 1:1. But if I clip my mic IRL by speaking loud, I can't clip the voice changer so feels little unnatural in this area. @deiteris
